token_keys/README.md

```diff
@@ -0,0 +1,2 @@
+# Public/Private Keys for Production
+This directory will contain the public and private keys used for signing the JWT's. After creation, these keys **must** be moved out of the project root to ensure they will not be served as static files by the server.
```

Comment:
typo at signing the JWT's. Should this be singing the JWTs?

Reply:
Yeah, I will update it. Nice catch!